README.md

```diff
@@ -15,26 +15,26 @@
 - [License](#license)
 
 ## Description
-The **Filename Inspector** GitHub Action is designed to ensure naming conventions in project files within specified repository directories. It scans for project files and reports any missing specified suffixes, helping maintain consistency and adherence to project standards. The tool is not limited to any programming language files; it scans file names and ignores extensions until they are used in filters.
+The **Filename Inspector** GitHub Action is designed to ensure naming conventions in project files within specified repository directories. It scans for project files and reports any missing specified file name patterns, helping maintain consistency and adherence to project standards. The tool is not limited to any programming language files; it scans file names and ignores extensions until they are used in filters.
 
 ## How It Works
-This action scans the specified `paths` for project files and checks if they end with the defined `suffixes` or contain them anywhere in the filename based on the `pattern_logic.` It reports the count of files not meeting the naming conventions, with options to fail the action if violations are found.
+This action scans the specified `paths` for project files and checks if their file names fit the `name_patterns.` It reports the count of files not meeting the naming conventions, with options to fail the action if violations are found.
 
 ## Inputs
-### `suffixes`
-- **Description**: List of suffixes that project files should have, separated by commas.
+### `name_patterns`
+- **Description**: List of file name [glob](https://en.wikipedia.org/wiki/Glob_(programming)) patterns that project files should have, separated by commas.
 - **Required**: Yes
-- **Default**: `*UnitTest.*,*IntegrationTest.*`
+- **Example**: `*UnitTest.*,*IntegrationTest.*`
 
 ### `paths`
-- **Description**: List of directories to include in the pattern check.
+- **Description**: List of directories to include in the glob pattern check, separated by commas.
 - **Required**: Yes
-- **Default**: `**/src/test/**/*.java,**/src/test/**/*.py`
+- **Example**: `**/src/test/**/*.java,**/src/test/**/*.py`
```

Comment:
```suggestion
- **Example**: `**/src/test/**/*.java,**/src/test/**`
```

Path should be just path to directories. Not with filenames as well.

Reply:
Fixed in commit 9b11bb975a4a693842b0858e0f373be0acef0833.